cleanslate/cleanslate/urls.py

```diff
@@ -33,6 +34,11 @@
     url(r'^doma/chore/(?P<pk>[-\w]+)/edit/$', edit_chore_deadline, name = 'edit-chore-deadline'),
     url(r'^doma/chore/create/$', create_chore, name = 'create-chore'),
     url(r'^doma/chore/(?P<pk>[-\w]+)/delete/$', delete_chore, name = 'delete-chore'),
+    url(r'^password_reset/$', auth_views.password_reset, name='password_reset'),
```

Comment:
These urls are already included on line 45

Reply:
Yeah, we have a lot of work to do with rearranging and removing unnecessary urls.